<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarteWidget</class>
 <widget class="QWidget" name="CarteWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="toolButton_2">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icones.qrc">
         <normaloff>:/icones/data/images/icons/zoom_out.png</normaloff>:/icones/data/images/icons/zoom_out.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icones.qrc">
         <normaloff>:/icones/data/images/icons/zoom_in.png</normaloff>:/icones/data/images/icons/zoom_in.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>50%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>200%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>400%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>800%</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_3">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icones.qrc">
         <normaloff>:/icones/data/images/icons/home.png</normaloff>:/icones/data/images/icons/home.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icones.qrc"/>
 </resources>
 <connections/>
</ui>
